tasks/create_target_vm/03_hosted_engine_final_tasks.yml

```diff
@@ -352,6 +352,11 @@
         register: vmremove_result
       - debug: var=vmremove_result
     when: local_vm_f.ansible_facts.ovirt_vms|length > 0
+  - name: Remove ovirt-engine-appliance rpm
+    yum:
+      name: ovirt-engine-appliance
```

Comment:
please add also rhvm-appliance for d/s

Reply:
on downstream spec file we already have:
Provides: ovirt-engine-appliance

so yum remove ovirt-engine-appliance will result in

Removing:
 rhvm-appliance